core_lib/src/graphics/bitmap/bitmapimage.cpp

```diff
@@ -157,6 +157,14 @@ BitmapImage BitmapImage::copy(QRect rectangle)
     if (rectangle.isEmpty() || mBounds.isEmpty()) return BitmapImage();
 
     QRect intersection2 = rectangle.translated(-mBounds.topLeft());
+
+    // If the region goes out of bounds, make sure the image is formatted in ARGB
+    // so that the area beyond the image bounds is transparent.
+    if (!mBounds.contains(rectangle) && !image()->hasAlphaChannel())
+    {
+        image()->convertTo(QImage::Format_ARGB32);
```

Comment:
This function was introduced in Qt 5.13.
https://doc.qt.io/qt-5/qimage.html#convertTo

Build check
https://ci.appveyor.com/project/chchwy/pencil2d-msvc/builds/39773062

Reply:
Will change to convertToFormat, thanks.